test/Log/OpusLogTest.php

```diff
@@ -0,0 +1,211 @@
+<?php
+/**
+ * This file is part of OPUS. The software OPUS has been originally developed
+ * at the University of Stuttgart with funding from the German Research Net,
+ * the Federal Department of Higher Education and Research and the Ministry
+ * of Science, Research and the Arts of the State of Baden-Wuerttemberg.
+ *
+ * OPUS 4 is a complete rewrite of the original OPUS software and was developed
+ * by the Stuttgart University Library, the Library Service Center
+ * Baden-Wuerttemberg, the Cooperative Library Network Berlin-Brandenburg,
+ * the Saarland University and State Library, the Saxon State Library -
+ * Dresden State and University Library, the Bielefeld University Library and
+ * the University Library of Hamburg University of Technology with funding from
+ * the German Research Foundation and the European Regional Development Fund.
+ *
+ * LICENCE
+ * OPUS is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU General Public License as published by the Free Software
+ * Foundation; either version 2 of the Licence, or any later version.
+ * OPUS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details. You should have received a copy of the GNU General Public License
+ * along with OPUS; if not, write to the Free Software Foundation, Inc., 51
+ * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ *
+ * @category    Test
+ * @package     Opus
+ * @author      Kaustabh Barman <barman@zib.de>
+ * @copyright   Copyright (c) 2020, OPUS 4 development team
+ * @license     http://www.gnu.org/licenses/gpl.html General Public License
+ */
+
+namespace OpusTest\Log;
+
+use Opus\Log\LogService;
+
+class OpusLogTest extends \PHPUnit_Framework_TestCase
+{
+    private $opusLog;
+
+    private $tempFolder;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $tempFolder = $this->createTempFolder();
+        $this->tempFolder = $tempFolder;
+        $this->createFolder('log');
+
+        $logService = LogService::getInstance();
```

Comment:
Are we testing the `LogService` here? Do we need the `LogService` here?

Reply:
If we want to use just the `Log` class object, we will have to take care of the format, file and priority separately. Or we can use some functions of `LogService` like `prepareFormat()`,  `getDefaultPriority()`, etc. but then we will be again using `LogService`.

Reply:
Why? What are we testing?

Reply:
Not using LogService anymore.